### PR TITLE
[master] Trigger manual DARC update

### PR DESCRIPTION
**Pull request by wtgodbe**
This is the result of triggering a manual DARC update, which we'll need since we were previously missing a subscription from dotnet/standard - this means the netstandard.dll we publish to Core-Sdk doesn't have the new ns2.1 types from the past ~1 month, e.g. `IAsyncEnumerable`. I added the subscription, so this should flow automatically from now on.

CC @mmitche @terrajobst @dsplaisted @danmosemsft

### CODE REVIEW

**Comment by wtgodbe** [comment]
> Microsoft.CSharp.Core.targets(58,5): error : Process is terminating due to StackOverflowException. [/Users/vsts/agent/2.147.1/work/1/s/src/System.Collections.Concurrent/src/System.Collections.Concurrent.csproj]

Same thing is happening in https://github.com/dotnet/corefx/pull/35571, looking now

**Comment by wtgodbe** [comment]
Same thing also happened in https://github.com/dotnet/corefx/pull/35378, so it seems related to Core-Setup

**Comment by wtgodbe** [comment]
Scoping this to just NetStandard.Library for now while I look into the error in System.Collections.Concurrent

**Comment by wtgodbe** [comment]
CC @terrajobst for the APICompat baselines - if the changes look good, I'll port to https://github.com/dotnet/corefx/pull/35571 as well

**Comment by wtgodbe** [comment]
Looks like the changes to UAP were bad, I'll look into why

**Comment by ericstj** [comment]
Those errors for UAP look bogus to me.  `TypeCannotChangeClassification` this isn't something we'd do.  The refs for UAP should have these defined correctly.  According to https://github.com/dotnet/corefx/commit/adf29843831d7c731f5fc72435bc4cbf979cdb55#diff-9b0d0fe2c10c3dbfd8e27a659830aad0R12 this might be APICompat choosing the wrong core assembly, if that's so we should fix it, potentially through explicit declaration via commandline parameter.

**Comment by wtgodbe** [comment]
> D:\a\1\s\src\shims\ApiCompat.proj(68,5): error : MembersMustExist : Member 'System.Runtime.CompilerServices.RuntimeHelpers.GetSubArray<T>(T[], System.Range)' does not exist in the implementation but it does exist in the contract.
D:\a\1\s\src\shims\ApiCompat.proj(68,5): error : MembersMustExist : Member 'System.Runtime.InteropServices.Marshal.GenerateGuidForType(System.Type)' does not exist in the implementation but it does exist in the contract.
D:\a\1\s\src\shims\ApiCompat.proj(68,5): error : MembersMustExist : Member 'System.Runtime.InteropServices.Marshal.GenerateProgIdForType(System.Type)' does not exist in the implementation but it does exist in the contract.
D:\a\1\s\src\shims\ApiCompat.proj(68,5): error : MembersMustExist : Member 'System.Runtime.InteropServices.Marshal.GetComObjectData(System.Object, System.Object)' does not exist in the implementation but it does exist in the contract.
D:\a\1\s\src\shims\ApiCompat.proj(68,5): error : MembersMustExist : Member 'System.Runtime.InteropServices.Marshal.GetHINSTANCE(System.Reflection.Module)' does not exist in the implementation but it does exist in the contract.
D:\a\1\s\src\shims\ApiCompat.proj(68,5): error : MembersMustExist : Member 'System.Runtime.InteropServices.Marshal.GetIDispatchForObject(System.Object)' does not exist in the implementation but it does exist in the contract.
D:\a\1\s\src\shims\ApiCompat.proj(68,5): error : MembersMustExist : Member 'System.Runtime.InteropServices.Marshal.GetTypedObjectForIUnknown(System.IntPtr, System.Type)' does not exist in the implementation but it does exist in the contract.
D:\a\1\s\src\shims\ApiCompat.proj(68,5): error : MembersMustExist : Member 'System.Runtime.InteropServices.Marshal.SetComObjectData(System.Object, System.Object, System.Object)' does not exist in the implementation but it does exist in the contract.

Got these errors for uapaot, looking into it now

File: src/shims/ApiCompatBaseline.uapaot.netstandard.txt [hunk 1]
```diff
@@ -3,6 +3,14 @@ TypeCannotChangeClassification : Type 'System.RuntimeArgumentHandle' is a 'ref s
 TypeCannotChangeClassification : Type 'System.TypedReference' is a 'ref struct' in the implementation but is a 'struct' in the contract.
 TypeCannotChangeClassification : Type 'System.IO.FileAttributes' is a 'class' in the implementation but is a 'struct' in the contract.
 Compat issues with assembly netstandard:
+MembersMustExist : Member 'System.Runtime.CompilerServices.RuntimeHelpers.GetSubArray(T[], System.Range)' does not exist in the implementation but it does exist in the contract.
```
Comment (ericstj) [inline src/shims/ApiCompatBaseline.uapaot.netstandard.txt]: Looks like you're missing the '<T>' from this method signature.
```
MembersMustExist : Member 'System.Runtime.CompilerServices.RuntimeHelpers.GetSubArray<T>(T[], System.Range)' does not exist in the implementation but it does exist in the contract.
```

File: src/shims/ApiCompatBaseline.uapaot.netstandardOnly.txt [hunk 2]
```diff
@@ -59,13 +59,14 @@ CannotRemoveBaseTypeOrInterface : Type 'System.Reflection.Emit.TypeBuilder' does
...
+CannotChangeAttribute : Attribute 'System.AttributeUsageAttribute' on 'System.Xml.Serialization.XmlAnyAttributeAttribute' changed from '[AttributeUsageAttribute(AttributeTargets.Field | AttributeTargets.Parameter | AttributeTargets.Property | AttributeTargets.ReturnValue)]' in the contract to '[AttributeUsageAttribute(AttributeTargets.Field | AttributeTargets.Parameter | AttributeTargets.Property | AttributeTargets.ReturnValue, AllowMultiple=false)]' in the implementation.
+CannotChangeAttribute : Attribute 'System.AttributeUsageAttribute' on 'System.Xml.Serialization.XmlNamespaceDeclarationsAttribute' changed from '[AttributeUsageAttribute(AttributeTargets.Field | AttributeTargets.Parameter | AttributeTargets.Property | AttributeTargets.ReturnValue)]' in the contract to '[AttributeUsageAttribute(AttributeTargets.Field | AttributeTargets.Parameter | AttributeTargets.Property | AttributeTargets.ReturnValue, AllowMultiple=false)]' in the implementation.
+MembersMustExist : Member 'System.Runtime.CompilerServices.RuntimeHelpers.GetSubArray(T[], System.Range)' does not exist in the implementation but it does exist in the contract.
```
Comment (ericstj) [inline src/shims/ApiCompatBaseline.uapaot.netstandardOnly.txt]: Missing here too.

File: src/shims/ApiCompatBaseline.uapaot.netstandard.txt (hunk 1 above)
Comment (wtgodbe) [reply src/shims/ApiCompatBaseline.uapaot.netstandard.txt]: Weird, the original source I copied this from didn't have the `<T>`: https://github.com/dotnet/corefx/pull/35572#issuecomment-467565178. I'll fix & update

**Comment by wtgodbe** [comment]
Arm64 failure looks to be related to the new queue, will re-run once the rest of the job finishes

**Comment by wtgodbe** [comment]
As per discussions with @safern & @MattGal, Linux/Mac issues are with the Test Queues, not due to the code changes, so I'm merging this now

**Pull request merged**